packages/release-mastermind/types/index.d.ts

```diff
@@ -3,0 +1,266 @@
+  configJSON: Runners
+  showLogs: boolean
+  dryRun: boolean
+  fillEmpty: boolean
+  skipDelete: boolean
+  repo?: Repo
+}
+
+export interface Runners {
+  labels?: Labels
+  runners: Config[]
+}
+
+export interface Config {
+  root: string
+  versioning: {
+    source: VersionSource
+    type?: VersionType
+  }
+  retryLimit?: number
+  prereleaseName?: string
+  labels?: { [key: string]: string }
+  sharedConfig?: SharedConfig
+  pr?: PullRequestConfig
+  issue?: IssueConfig
+  project?: ProjectConfig
+  schedule?: ScheduleConfig
+}
+
+/**
+ * Config types
+ */
+
+export type VersionSource = 'node' | 'milestones' | string
+export type VersionType = 'SemVer'
+
+export interface SharedConditions {
+  requires: number
+  conditions: Condition[]
+}
+
+export interface Label {
+  name: string
+  description: string
+  color: string
+}
+
+export interface Labels {
+  [key: string]: Label
+}
+
+export interface PRConditionConfig {
+  requires: number
+  conditions: PRCondition[]
+}
+
+export interface IssueConditionConfig {
+  requires: number
+  conditions: IssueCondition[]
+}
+
+export interface ProjectConditionConfig {
+  requires: number
+  conditions: ProjectCondition[]
+}
+
+export interface ScheduleConditionConfig {
+  requires: number
+  conditions: ScheduleCondition[]
+}
+
+export interface PullRequestConfig extends SharedConfig {
+  assignProject?: AssignProject[]
+  automaticApprove?: AutomaticApprove
+  manageRelease?: Release
+  duplicateHotfix?: { [title: string]: DuplicateHotfix }
+  syncRemote?: SyncRemote[]
+}
+
+export interface IssueConfig extends SharedConfig {
+  assignProject?: AssignProject[]
+  createBranch?: { [label: string]: CreateBranch }
+}
+
+export interface ProjectConfig extends SharedConfig {
+  syncRemote?: ExProjects[]
+  openBranch?: ProjectCreateBranch
+  assignMilestone?: { [milestone: string]: Milestones }
+}
+
+export interface ScheduleConfig extends SharedConfig {}
+
+export type SharedConfigIndex =
+  | 'ref'
+  | 'enforceConventions'
+  | 'labels'
+  | 'stale'
+
+export interface SharedConfig {
+  ref?: string
+  enforceConventions?: EnforceConventions
+  stale?: Stale
+  labels?: {
+    [key: string]:
+      | IssueConditionConfig
+      | ProjectConditionConfig
+      | PRConditionConfig
+      | ScheduleConditionConfig
+      | SharedConditions
+  }
+}
+
+export interface SharedConventionConditions {
+  requires: number
+  conditions: Condition[] | string
+}
+export interface SharedConventionsConfig extends SharedConventionConditions {
+  failedComment: string
+  contexts?: string[]
+}
+
+export interface CreateBranch {
+  branchPrefix?: string
+  branchSuffix?: string
+  branchName: 'title' | 'short' | 'number'
+}
+
+export interface EnforceConventions {
+  onColumn?: Column[]
+  commentHeader?: string
+  commentFooter?: string
+  moveToColumn?: string
+  conventions: SharedConventionsConfig[]
+}
+
+export interface Stale {
+  staleLabel: string
+  stale?: StaleConfig
+  abandoned?: AbanondedConfig
```

Comment:
Codacy has a fix for the issue: [Expected a semicolon.](https://app.codacy.com/gh/Videndum/action-masterminds/pullRequest?prid=7872005)
```suggestion
  abandoned?: AbanondedConfig;
```